blocks/api/paste.js

```diff
@@ -76,7 +78,9 @@ export function normaliseToBlockLevelNodes( nodes ) {
 }
 
 export default function( nodes ) {
-	return normaliseToBlockLevelNodes( nodes ).map( ( node ) => {
+	const prepare = compose( [ normaliseToBlockLevelNodes, gDocs, stripAttributes ] );
```

Comment:
Ah, sure. I guess the idea of a separate function comes from TinyMCE also running processors which strip identifying content, and running the test function even before that.